.github/workflows/build.yml

```diff
@@ -49,8 +49,7 @@ jobs:
         bundle install --jobs 4 --retry 3
         sudo npm --unsafe-perm rebuild node-sass
         NODE_ENV=test bundle exec rails webpacker:compile
-        bundle exec rails db:create
-        bundle exec rails db:schema:load
+        bundle exec rails db:setup
```

Comment:
This will run the seeds.rb file as well - is this what you want?

Reply:
[yep](https://github.com/unboxed/bops/pull/384/commits/22c4e1076d8706c1af179aa8f56cf38967c8d8cc)